Stormware's Pohoda schema ready

Vitexus opened the pull request:
Huge & Complex schema https://www.stormware.cz/xml/schema/all_schema_ver2.zip

accountancy.xsd
accountingFormOfPayment.xsd
accountingSalesVouchers.xsd
accountingunit.xsd
actionPrice.xsd
activity.xsd
addressbook.xsd
advancePartFulfilment.xsd
archive.xsd
automaticLiquidation.xsd
balance.xsd
bankAccount.xsd
bank.xsd
cashRegister.xsd
category.xsd
centre.xsd
classificationVAT.xsd
contract.xsd
data-package.xsd
data.xsd
discount.xsd
documentresponse.xsd
enquiry.xsd
establishment.xsd
filter.xsd
GDPR.xsd
groupStocks.xsd
individualPrice.xsd
intDoc.xsd
intParam.xsd
inventoryLists.xsd
invoice.xsd
isdoc.xsd
liquidationWithoutLink.xsd
list_activity.xsd
list_addBook.xsd
list_centre.xsd
list_contract.xsd
list_stock.xsd
list.xsd
lock.xsd
mKasa.xsd
movement.xsd
numericalSeries.xsd
offer.xsd
order.xsd
parameter.xsd
payment.xsd
prevodka.xsd
prijemka.xsd
print.xsd
prodejka.xsd
productRequirement.xsd
recyclingContrib.xsd
registrationNumber.xsd
response.xsd
rulesPairing.xsd
sendEET.xsd
service.xsd
stock.xsd
storage.xsd
store.xsd
supplier.xsd
type.xsd
userAgenda.xsd
voucher.xsd
vydejka.xsd
vyroba.xsd

Vitexus committed:
Stormware's Pohoda schema ready

goetas commented:
could you add tests for your changes?

Vitexus commented:
> could you add tests for your changes?

If my proposed addition to the library is okay, I will be happy to add the phpunit tests as well.

goetas commented:
your changes are reasonable. i'm surprised it wasn't already implemented that way

Vitexus committed:
#191 Add unit tests for PhpConverter class

Introduces PHPUnit-based tests for the PhpConverter class,
covering methods like convert, visitElementDef, isSkip, and visitType.
Ensures functionality aligns with expected behavior when working
with XSD schemas and their corresponding PHP class generation.
